community/security/src/main/java/org/neo4j/server/security/auth/AuthenticationStrategy.java

```diff
@@ -19,6 +19,8 @@
  */
 package org.neo4j.server.security.auth;
 
+import org.neo4j.kernel.api.security.AuthenticationResult;
```

Comment:
Yes, AuthenticationResult moved from org.neo4j.server.security.auth to org.neo4j.kernel.api.security